File: replace.py
import re
import os

file_path = 'src/features/monitoring-detection/MonitoringDetectionClient.tsx'
with open(file_path, 'r', encoding='utf-8') as f:
    content = f.read()

replacements = [
    (r"font-\['Montserrat'\]", "font-montserrat"),
    (r"bg-\[#eff4eb\]", "bg-greenLight"),
    (r"text-\[#2B4D1A\]", "text-greenDark"),
    (r"border-\[#dbe6d4\]", "border-greenLightHover"),
    (r"bg-\[#fff9e6\]", "bg-yellowLight"),
    (r"text-\[#856404\]", "text-yellowDark"),
    (r"border-\[#ffeeba\]", "border-yellowLightHover"),
    (r"bg-\[#47802B\]", "bg-greenNormal"),
    (r"border-black/10", "border-lineSoft"),
    (r"text-\[#243620\]", "text-inkDark"),
    (r"bg-\[#2B4D1A\]", "bg-greenDark"),
    (r"hover:bg-\[#203a13\]", "hover:bg-greenDarker"),
    (r"bg-\[#E9F2FF\]", "bg-surface"),
    (r"text-\[#222222\]", "text-ink"),
    (r"bg-\[#f8faf7\]", "bg-canvas"),
    (r"bg-\[#FBF4D7\]", "bg-yellowLight"),
    (r"bg-\[#E3ECDF\]", "bg-greenLight"),
    (r"border-\[#2B4D1A\]/50", "border-greenDark/50"),
    (r"bg-\[#F6FAF3\]", "bg-surface"),
    (r"bg-\[#356020\]", "bg-moss"),
    (r"border-\[#D1D9CA\]", "border-line"),
    (r"bg-\[#C7DABA\]", "bg-mossPale"),
    (r"bg-\[#396622\]", "bg-greenNormal"),
    (r"bg-\[#fbfcfa\]", "bg-canvas"),
    (r"text-black/50", "text-muted"),
    (r"text-black/35", "text-muted/70"),
    (r"(?<!-)text-black(?!\w)", "text-ink")
]

for old, new in replacements:
    content = re.sub(old, new, content)

with open(file_path, 'w', encoding='utf-8') as f:
    f.write(content)

print('Replacements completed.')

File: replace.js
const fs = require('fs');

const filePath = 'src/features/monitoring-detection/MonitoringDetectionClient.tsx';
let content = fs.readFileSync(filePath, 'utf8');

const replacements = [
  [/font-\['Montserrat'\]/g, 'font-montserrat'],
  [/bg-\[#eff4eb\]/g, 'bg-greenLight'],
  [/text-\[#2B4D1A\]/g, 'text-greenDark'],
  [/border-\[#dbe6d4\]/g, 'border-greenLightHover'],
  [/bg-\[#fff9e6\]/g, 'bg-yellowLight'],
  [/text-\[#856404\]/g, 'text-yellowDark'],
  [/border-\[#ffeeba\]/g, 'border-yellowLightHover'],
  [/bg-\[#47802B\]/g, 'bg-greenNormal'],
  [/border-black\/10/g, 'border-lineSoft'],
  [/text-\[#243620\]/g, 'text-inkDark'],
  [/bg-\[#2B4D1A\]/g, 'bg-greenDark'],
  [/hover:bg-\[#203a13\]/g, 'hover:bg-greenDarker'],
  [/bg-\[#E9F2FF\]/g, 'bg-surface'],
  [/text-\[#222222\]/g, 'text-ink'],
  [/bg-\[#f8faf7\]/g, 'bg-canvas'],
  [/bg-\[#FBF4D7\]/g, 'bg-yellowLight'],
  [/bg-\[#E3ECDF\]/g, 'bg-greenLight'],
  [/border-\[#2B4D1A\]\/50/g, 'border-greenDark/50'],
  [/bg-\[#F6FAF3\]/g, 'bg-surface'],
  [/bg-\[#356020\]/g, 'bg-moss'],
  [/border-\[#D1D9CA\]/g, 'border-line'],
  [/bg-\[#C7DABA\]/g, 'bg-mossPale'],
  [/bg-\[#396622\]/g, 'bg-greenNormal'],
  [/bg-\[#fbfcfa\]/g, 'bg-canvas'],
  [/text-black\/50/g, 'text-muted'],
  [/text-black\/35/g, 'text-muted\/70'],
  [/(?<!\w)text-black(?!\w)/g, 'text-ink']
];

for (const [regex, replacement] of replacements) {
  content = content.replace(regex, replacement);
}

fs.writeFileSync(filePath, content, 'utf8');
console.log('Replacement completed.');
